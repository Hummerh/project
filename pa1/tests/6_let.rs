let x = -5;
// x is 5!
