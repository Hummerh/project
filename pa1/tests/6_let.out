let
ID(x)
=
-
LITDEC(5)
;
